content/00.build/05.start-coding/10.zksync-101/50.paymaster.md

```diff
@@ -57,6 +57,10 @@ The paymaster smart contract code is provided "as-is" without any express or imp
 
 ---
 
+::callout{icon="i-heroicons-information-circle" color="blue"}
+If you have not started up a local node yet from the Getting started page, run `zksync-cli dev start`.
+::
```

Comment:
does this one not need the "Make sure you are running the "In memory node" option." line

Reply:
oh good catch, let me update that.